website/docs/developer/deployment-guide/deployment.md

```diff
@@ -1,8 +1,51 @@
 ---
 id: deployment
-title: Developer Deployment
+title: Deployment Information
 description: Provides developer information for ORCA code deployment.
 ---
 
-This document is currently under construction.
+## Introduction to ORCA
+The Operational Recovery Cloud Archive (ORCA) provides 
+a baseline solution for creating, and managing 
+operational backups in the cloud. In addition, best
+practices and recovery code that manages common 
+baseline recovery scenarios is also maintained.
+
+
+## Deploying ORCA
+The following sections provide information on how to
+deploy ORCA with or along-side Cumulus.
+### Setup Deployment Environment
+In order to deploy ORCA into an NGAP environment with 
+Cumulus, a deployment environment needs be created.
+
+Visit [Creating a Deployment Environment](setting-up-deployment-environment.md)
+to view the instructions for setting up your ORCA deployment environment.
+
+### ORCA Archive Bucket
+ORCA maintains a cloud ready backup of science and 
+ancillary data in an S3-IA bucket and utilizes AWS 
+bucket transition policies to store the data in S3 
+Glacier for the long term. The ORCA archive bucket can 
+live in any NGAP Organizational Unit (OU).
```

Comment:
Should we add a couple of these terms to the glossary.md?

Reply:
will put a new card into the backlog for OU and NGAP